src/main/java/de/tum/in/www1/artemis/web/rest/LectureResource.java

```diff
@@ -277,13 +277,15 @@ public ResponseEntity<Boolean> ingestLectures(@PathVariable Long courseId, @Requ
             if (lectureToIngest.isPresent()) {
                 Set<Lecture> lecturesToIngest = new HashSet<>();
                 lecturesToIngest.add(lectureToIngest.get());
-                return ResponseEntity.ok().body(lectureService.ingestLecturesInPyris(lecturesToIngest));
+                lectureService.ingestLecturesInPyris(lecturesToIngest);
+                return ResponseEntity.ok().body(true);
             }
             return ResponseEntity.badRequest()
                     .headers(HeaderUtil.createAlert(applicationName, "Could not send lecture to Iris, no lecture found with the provided id.", "idExists")).body(null);
 
         }
-        return ResponseEntity.ok().body(lectureService.ingestLecturesInPyris(course.getLectures()));
+        lectureService.ingestLecturesInPyris(course.getLectures());
+        return ResponseEntity.ok().body(true);
```

Comment:
Will we ever return false here? Otherwise it would be enough to return nothing and rely on the status code of the response